packages/server/src/workers/reindex.ts

```diff
@@ -45,7 +45,7 @@ export function initReindexWorker(config: MedplumServerConfig): void {
   queue = new Queue<ReindexJobData>(queueName, {
     ...defaultOptions,
     defaultJobOptions: {
-      attempts: 3,
+      attempts: 1,
```

Comment:
In the case that a job fails due to some connection error with Redis or similar, don't we maybe want to still retry?

A potential way to think about it:
From the perspective of `bullmq` job only truly fails if errors other than a cancellation occur...

Not sure if that makes sense but something to consider

Reply:
The way the job is written, all errors that would be thrown are caught by the worker and result in the `AsyncJob` being updated to `error` status.  There's no possibility for the job to be retried anyway, so I was just aligning this to that reality